internal/drivers/google/driver.go

```diff
@@ -367,6 +422,29 @@ func (p *config) getInstance(ctx context.Context, projectID, zone, name string)
 	})
 }
 
+func (p *config) getInstanceIP(i *compute.Instance) string {
+	instanceIP := ""
+	network := i.NetworkInterfaces[0]
+	if p.privateIP {
+		instanceIP = network.NetworkIP
+	} else {
+		instanceIP = network.AccessConfigs[0].NatIP
```

Comment:
Can we add a check that AccessConfig exists before trying to go a level deeper

Reply:
Done